data/json/items/magazine/50.json

```diff
@@ -27,5 +27,22 @@
     "reliability": 9,
     "reload_time": 130,
     "flags": [ "MAG_BULKY" ]
+  },
+  {
+    "id": "as50mag",
+    "type": "MAGAZINE",
+    "name": "AS50 Magazine"
```

Comment:
```suggestion
    "name": "AS50 Magazine",
```

Reply:
ahh thank you for the help, that's the problem im trying to avoid (my eyes can't even notice a missing comma there)